manifest-build-tools/application/update_rackhd_version.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python
```

Comment:
Can you help to figure out a better name ?
it will change the RackHD/RackHD/control file , and update the dependency components' debian package version.
if the filename change, then it's a :+1: